__tests__/middleware.test.ts

```
@@ -0,0 +1,354 @@
+  NextResponse: {
+    next: jest.fn(),
+    redirect: jest.fn(),
+  },
+}));
+
+const mockGetToken = getToken as jest.MockedFunction<typeof getToken>;
+const mockIsFeatureEnabledEdge = isFeatureEnabledEdge as jest.MockedFunction<typeof isFeatureEnabledEdge>;
+const mockNextResponse = NextResponse as jest.Mocked<typeof NextResponse>;
+
+describe('middleware', () => {
+  let mockRequest: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)